docs/cmdlets-reference/set-cohesityview.md

```diff
@@ -19,6 +19,17 @@ Returns the updated Cohesity View.
 Set-CohesityView -View $view
 ```
 
+### EXAMPLE 2
+Override global whitelist
+```
+$view = Get-CohesityView -ViewNames “Cohesity_View”
+$SubnetWhitelists = New-Object 'System.Collections.Generic.List[Cohesity.Model.Subnet]'
+$subnet =  New-Object -TypeName Cohesity.Model.Subnet("") -Property @{NetmaskIp4="255.255.255.0" Description="whitelist linux";Ip="10.2.146.241";NfsAccess=3; SmbAccess=2;NfsRootSquash=$False}
```

Comment:
How are other examples ? can you give a generic IP instead of this IP ?

Reply:
I will change it to some random/generic ip. Other examples are mostly single line and are not lengthy scripts. This example is a multi-line script since it needs the view model to be defined before using Set-CohesityView. Other cmdlets which use model arguments do not show how we define the model ex: https://cohesity.github.io/cohesity-powershell-module/#/cmdlets-reference/set-cohesitystoragedomain

Reply:
Cool